src/plots/cartesian/axes.js

```diff
@@ -1042,7 +1042,8 @@ function autoTickRound(ax) {
 
         var maxend = Math.max(Math.abs(rng[0]), Math.abs(rng[1]));
         var rangeexp = Math.floor(Math.log(maxend) / Math.LN10 + 0.01);
-        if(Math.abs(rangeexp) > ax.minexponent) {
+        var minexponent = ax.minexponent === undefined ? 3 : ax.minexponent;
```

Comment:
Why do we need to check for `ax.minexponent === undefined` here?
Which test is failing without it?

Reply:
I can go back and get the names. It looks like there are a number of places in the code which create mock `axes` objects in a way that leaves `minexponent` undefined.

Reply:
For example, look at

https://github.com/plotly/plotly.js/pull/5121/files#diff-f6d5ed25a5d6f57c70063e1e1bbc407dR1526